keepalived/vrrp/vrrp_scheduler.c

```diff
@@ -783,17 +783,23 @@ vrrp_lower_prio_gratuitous_arp_thread(thread_t * thread)
 	return 0;
 }
 
+void
+vrrp_log_effective_priority(vrrp_t *vrrp)
+{
+	log_message(LOG_INFO, "VRRP_Instance(%s) Effective priority = %d",
+		    vrrp->iname, vrrp->effective_priority);
+}
+
 /* Set effective priorty, issue message on changes */
 void
 vrrp_set_effective_priority(vrrp_t *vrrp, int new_prio)
 {
 	if (vrrp->effective_priority == new_prio)
 		return;
 
-	log_message(LOG_INFO, "VRRP_Instance(%s) Effective priority = %d",
-		    vrrp->iname, new_prio);
```

Comment:
This, followed by the call of vrrp_log_effective_priority() below will produce output like:
```
VRRP_Instance(VI_1) Effective priority = 20
VRRP_Instance(VI_1) Effective priority = 30
```
which is perhaps slightly confusing.

Would it be better to change this to:
```
log_message(LOG_INFO, "VRRP_Instance(%s) Changing Effective priority from %d to %d",
                       vrrp->iname, vrrp->effective_priority, new_prio);
```
and not calling `vrrp_log_effective_priority()` at line 802 below?